repository/Seaside-Zinc-Core.package/ZnZincServerAdaptor.class/instance/configureServerForBinaryReading.st

```diff
@@ -0,0 +1,7 @@
+initialize-release
```

Comment:
In Pharo at least, they got rid of this protocol.

Reply:
Not everywhere: ZnEntity (e.g.) still has it. But I agree, so I changed it here.